src/session4/challenge/challenge5.java

```diff
@@ -0,0 +1,20 @@
+package session4.challenge;
+
+public class challenge5 {
+
+    //5. Write a Java program to print the numbers from 1 to 10 using a for loop. As a next step, modify your program to print only the even numbers from 1 to 10.
+
+    public static void main(String[] args) {
+        for (int i = 1; i <= 10; i++) {
```

Comment:
Please try to use more descriptive variabiles; in this case, we can name it `index` instead of `i`.